Content.Shared/Interaction/InteractHand.cs

```diff
@@ -36,4 +36,23 @@ public InteractHandEvent(EntityUid user, EntityUid target)
             Target = target;
         }
     }
+
+    public sealed class InteractNoHandEvent : HandledEntityEventArgs, ITargetedInteractEventArgs
+    {
+        /// <summary>
+        ///     Entity that triggered the interaction.
+        /// </summary>
+        public EntityUid User { get; }
+
+        /// <summary>
+        ///     Entity that was interacted on.
+        /// </summary>
+        public EntityUid Target { get; }
+
+        public InteractNoHandEvent(EntityUid user, EntityUid target)
+        {
+            User = user;
+            Target = target;
+        }
+    }
 }
```

Comment:
Isn't this just an InteractHandEvent? You're also doing the raycast for it too so it looks identical.

Reply:
The InteractHandEvent requires that the entity interacting has a hand. I was unsure about modifying that, as a lot of things seems to use that interaction. Instead, i opted to create a separate interaction for hands-less entities.